module/platform/NUSense/HardwareIO/src/HardwareIO.cpp

```diff
@@ -119,6 +119,11 @@ namespace module::platform::NUSense {
             // Battery data
             sensors->battery = 0;  // not yet implemented
 
+            // Log the latest dispatch from NUSense; only very rare exceptions should be dispatched.
+            if (data.dispatch.size() > 3) {
+                log<NUClear::WARN>("Dispatch from NUSense: " + data.dispatch);
+            }
+
```

Comment:
I think this should just be a separate message, rather than baked into the mega message. It's making that message very hard to work with.

For any exceptional error, just send a "log" message of some kind, then you can on trigger it in here and do something with it

Reply:
Will remove the `dispatch` field and move it to another message in a different PR.